packages/engine/Source/Core/PolygonGeometryLibrary.js

```diff
@@ -323,6 +323,11 @@ PolygonGeometryLibrary.subdivideRhumbLine = function (
   const c1 = ellipsoid.cartesianToCartographic(p1, scratchCartographic1);
   const rhumb = new EllipsoidRhumbLine(c0, c1, ellipsoid);
 
+  if (rhumb.surfaceDistance === 0) {
```

Comment:
Can we use an [`equalsEpsilon`](https://cesium.com/learn/ion-sdk/ref-doc/Math.html#.equalsEpsilon) check?

Or maybe better yet, we can return earlier as long as the surface distance is less then or equal to `minDistance`, right?